temporalcli/commandsgen/commands.yml

```diff
@@ -1071,12 +1074,14 @@ commands:
         type: string-enum[]
         description: Search Attribute type.
         enum-values:
-          - Text
-          - Keyword
-          - Int
-          - Double
-          - Bool
-          - Datetime
+          - text
+          - keyword
+          - int
+          - double
+          - bool
+          - datetime
+          - keyword-list
```

Comment:
These values represent actual, documented constants at https://docs.temporal.io/visibility#supported-types. Not sure the CLI's desire for enum consistency should override Temporal's desire for consistency for all uses of these constants in all environments.

Reply:
gotcha, yeah while going through this exercise I was wondering if there were certain constants that were beyond the CLI and take precedence over this idea for consistency

Reply:
I do not have a strong opinion, it is just worth noting for this enumerate specifically we document certain string terms globally